index.html

```diff
@@ -647,6 +647,28 @@
   </div>
 </div>
 <!-- ends Burn in timecode -->
+<!-- Check FFv1 fixity -->
+<span data-toggle="modal" data-target=".check_FFv1_fixity"><button type="button" class="btn btn-default" data-toggle="tooltip" data-placement="bottom" title="This decodes your video and verifies the internal crc checksums">Check FFv1 fixity</button></span>
+<div class="modal fade check_FFv1_fixity" tabindex="-1" role="dialog" aria-labelledby="myLargeModalLabel">
+  <div class="modal-dialog modal-lg">
+    <div class="modal-content">
+      <div class="well">
+        <h3>Check FFv1 Version 3 fixity</h3>
+        <p><code>ffmpeg -i <i>input_file</i> -f null -</code></p>
```

Comment:
My understanding is that frame crcs are not available in ffv1 version 1. Using zzuf, I haven't been able to produce any crc mismatches with ffv1 version 1.